packs/zookeeper/actions/non_blocking_lease.py

```diff
@@ -0,0 +1,49 @@
+from datetime import timedelta
+from kazoo.client import KazooClient
+from socket import gethostname
+from st2actions.runners.pythonrunner import Action
+import sys
```

Comment:
Fixed. Assuming you want alphanumeric ordering within groups, which PEP-8 doesn't actually specify. https://www.python.org/dev/peps/pep-0008/#imports :)